app/views/students/_form.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1>Students#_form</h1>
+<p>Find me in app/views/students/_form.html.erb</p>
+
+<div class="form d-flex justify-content-center">
+  <%= form_for(@student) do |student| %>
```

Comment:
good job using the helpers to create a form 👍 

minor comment:
`form_for` is soft-deprecated to get more details about it to follow this [link](https://guides.rubyonrails.org/form_helpers.html#using-form-tag-and-form-for)

you could use `form_with` more info [here](https://guides.rubyonrails.org/form_helpers.html#binding-a-form-to-an-object)

Reply:
thanks! i will look that info